File: RETO272.cpp
//Sergio Ruescas
#include <iostream>
using namespace std;

void ToBase6(int number)
{
    int divisor, resto;
    if ( number < 6 )
    {
        printf("%d", number);
    }
    else
    {
        divisor = number / 6;
        resto = number%6;
        
        ToBase6(divisor);
        
        printf("%d", resto);
    }
    
    
}

int main()
{
    int number, count;
    scanf("%d",&count);
    for (int i = 0; i < count; i++)
    {
        scanf("%d",&number);
        ToBase6(number);
        printf("\n");
    }
    
    return 0;
}

File: RETO272.java
//Sergio Ruescas
import  java.util.Scanner;

public class RETO172
{
    public static void ToBase6(int number)
    {
        int divisor, resto;
        if ( number < 6 )
        {
            System.out.printf("%d", number);
        }
        else
        {
            divisor = number / 6;
            resto = number%6;
            
            ToBase6(divisor);
            
            System.out.printf("%d", resto);
        }
        
        
    }
    public static void main(String[] text)
    {
        Scanner sc = new Scanner(System.in);
        int cases = sc.nextInt();
        for (int i = 0; i < cases; i++)
        {
            int number = sc.nextInt();
            ToBase6(number);
            System.out.println();
        }
        sc.close();
    }
        
}
    

